service/resource/configmap/desired_test.go

```diff
@@ -221,9 +221,9 @@ func Test_Resource_ConfigMap_GetDesiredState(t *testing.T) {
 						Scheme:  "https",
 						HTTPClientConfig: config.HTTPClientConfig{
 							TLSConfig: config.TLSConfig{
-								CAFile:             "/certs/xa5ly/ca.pem",
-								CertFile:           "/certs/xa5ly/crt.pem",
-								KeyFile:            "/certs/xa5ly/key.pem",
+								CAFile:             "/certs/xa5ly-ca.pem",
```

Comment:
I also decided to flatten the certificate filesystem layout. Should be marginally easier to manage.